gr-utils/modtool/support/_gr_modtool

```diff
@@ -0,0 +1,25 @@
+#compdef gr_modtool
+#autoload
+
```

Comment:
Note that I find this awesome - thanks for dropping by! This is super helpful.

* `#autoload`: gotcha, will be removed.
* `_gnuradio` instead of `_gr_modtool`: I must admit I have very thin understanding what you mean with `$service` here. I followed https://github.com/zsh-users/zsh-completions/blob/master/zsh-completions-howto.org#telling-zsh-which-function-to-use-for-completing-a-command ; what I understand now is that I'd start the file with `#compdef _gnuradio`, and have a `compdef _gnuradio gr_modtool gnuradio-config-info gr_plot_iq` (and so on) at the end.<br> But how does my script then know whether I'm completing for `gr_modtool` or `gnuradio-config-info`?

Reply:
You start the file with `#compdef gr_modtool gnuradio-config-info gr_plot_iq`. No need to list `_gnuradio` in there, it's only with the `compdef` command that you'd say that. That would be used as the filename, though. Then in the function body you can switch on `$service`, typically:

```
    case $service in
      gr_modtool)
        ...
      ;;
      gr_plot_iq) ... ;;
    esac
```